test_accounts.md

```diff
@@ -1,16 +1,26 @@
 # Sandbox Test Accounts for VA Lighthouse APIs
 
-Lighthouse provides test accounts and data you can use to make sure your integration works as planned. These test accounts are specific to each API. They can be used to log in to the ID.me sandbox environment, which is connected to sandbox-api.va.gov.
+Lighthouse provides test accounts and data you can use to make sure your integration works as planned. These test accounts are specific to each API and login method. Both ID.me and Login.gov connect to sandbox-api.va.gov. If you use Login.gov, you’ll need an application that generates multi-factor authentication codes, such as Google Authenticator or Authy.
 
-To log in to the sandbox environment with a test account, make sure you:
+**Do not modify any of the attributes for these test accounts, including passwords.** If you need to use one of these accounts for a different use case, [contact us](https://developer.va.gov/support/contact-us) on our developer portal.
+
+## Logging in with ID.me
+
+To log in to the sandbox environment using ID.me, make sure you:
 
   1. Choose an account that is specific to the API you want to test.
-  2. Use ID.me to sign in to the sandbox environment.
-  3. **Don’t change any preselected answers** when asked about receiving an authentication code. Just click “Continue” to go to the next step.
+  2. Select **ID.me** to sign in to the sandbox environment.
+  3. Enter the ID.me username and password. The password for all ID.me test accounts is: **Password1234!**
+  4. **Don’t change any preselected answers** when asked about receiving an authentication code. Just click “Continue” to go to the next step.
 
-**Do not modify any of the attributes for these test accounts, including passwords.** If you need to use one of these accounts for a different use case, [contact us](https://developer.va.gov/support/contact-us) on our developer portal.
+## Logging in with Login.gov
+
+To log in to the sandbox environment using Login.gov, make sure you:
 
-Find out more about each API by selecting one below.
+  1. Choose an account that is specific to the API you want to test.
+  2. Select **Login.gov** to sign in to the sandbox environment.
+  3. Enter the Login.gov username and password. The password for all Login.gov test accounts is: **Password12345!!!**
```

Comment:
Just double check to make sure the passwords are correct.

Reply:
This is correct.